.github/workflows/publish.yml

```diff
@@ -23,5 +23,38 @@ jobs:
         with:
           sdk: 3.4
 
+      - name: Download cwebp
+        run:
+          mkdir tmp
+          cd tmp
+
+          curl https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.4.0-windows-x64.zip -o windows-x64.zip
```

Comment:
Let's move the `1.4.0` to a variable (within the script) so it's set only in one place

Reply:
Could you check if I've done it properly?

Reply:
So I meant using a Bash variable, but this works too, maybe even better, because we have it at the top of the whole workflow

Reply:
ah okay